infrastructure/storage-provisioning/README.md

```diff
@@ -85,3 +85,25 @@ After that you can delete test with commands
 $ kubectl delete -f examples/wordpress.yaml
 $ kubectl delete -f examples/mysql.yaml
 ```
+
+## Rook NFS
+To use the user's personal storage we decided to use the NFS server provided by Rook-Ceph ([here](https://rook.io/docs/rook/v1.10/Storage-Configuration/NFS/nfs/)).
+
+With the `CephNFS` CRD we configure a NFS-Ganesha server to provide the access to the user's personal storage via NFS inside the cluster.
+
+A `StorageClass` is used for the dynamic provisioning of the user's storage. In this `StorageClass` is specified, via specific parameters for the Rook Operator, to create also a NFS share to which the user can later connect to via the NFS protocol.
+
+To create the necessary resources:
+```bash
+$ kubectl create -f nfs-manifests/nfs.yaml
+$ kubectl create -f nfs-manifests/storageclass.yaml
+```
+
+Be aware that to use this feature the system must be configured with:
+- Rook v1.9.1 or greather
+- Ceph v16.2.7 or greather
+
+### Using the NFS service
```

Comment:
I would say something about quotas here: is it guaranteed?
Just something "Per-user parameters such as disk quota can be specified in XXX".

Reply:
The parameter for the quota of the user is in the Tenant Operator. Is used for the PVC size.
The parameter is described in the operators/README.md with the other parameters.